lib/kracken/session_manager.rb

```diff
@@ -1,13 +1,27 @@
 # frozen_string_literal: true
 
+require 'redis'
+
 module Kracken
   module SessionManager
     def self.conn
-      @conn ||= if ENV["REDIS_SESSION_URL"].present?
-                  Redis.new(url: ENV["REDIS_SESSION_URL"])
-                else
-                  NullRedis.new
-                end
+      @conn ||=
+        begin
+          default_redis_options = { url: ENV['REDIS_SESSION_URL'] }
+          redis_options = default_redis_options.merge(Kracken.config.redis_options).compact
+
+          if redis_options.any?
+            Redis.new(**redis_options)
+          else
+            NullRedis.new
+          end
```

Comment:
I wonder if this should explicitly check if the `url` option is set? Depending on how apps set the redis options in development, I could see how it would be easy to set the options but not the URL and still expect the previous functionality (ie, `NullRedis`).